src/docfx/build/Build.cs

```diff
@@ -58,8 +58,6 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Erro
 
                 var (publishManifest, fileManifests, sourceDependencies) = await BuildFiles(context, docset, tocMap);
 
-                var saveGitHubUserCache = context.GitHubUserCache.SaveChanges(config);
```

Comment:
then docfx doesn't have the ablitity to update remote github user cache, this part will be done by Docs.Build including the merge logic?

Reply:
or my question is that in the future, the cahche will be per repo/branch?

---
In reply to: [293177576](https://github.com/dotnet/docfx/pull/4725#discussion_r293177576) [](ancestors = 293177576)